sc/src/main/java/uk/ac/ebi/atlas/experimentpage/SingleCellContentService.java

```diff
@@ -1,125 +1,55 @@
 package uk.ac.ebi.atlas.experimentpage;
 
-import javax.inject.Named;
+import com.google.common.collect.ImmutableList;
+import uk.ac.ebi.atlas.model.download.ExternallyAvailableContent;
+import uk.ac.ebi.atlas.model.experiment.Experiment;
+import uk.ac.ebi.atlas.model.experiment.baseline.BaselineExperiment;
+import uk.ac.ebi.atlas.trader.ExperimentTrader;
 
-//
-//import com.google.common.collect.ImmutableList;
-//import uk.ac.ebi.atlas.controllers.rest.experimentdesign.ExperimentDesignFile;
-//import uk.ac.ebi.atlas.experimentpage.differential.download.DifferentialSecondaryDataFiles;
-//import uk.ac.ebi.atlas.experimentpage.qc.RnaSeqQCReport;
-//import uk.ac.ebi.atlas.model.download.ExternallyAvailableContent;
-//import uk.ac.ebi.atlas.model.experiment.Experiment;
-//import uk.ac.ebi.atlas.model.experiment.baseline.BaselineExperiment;
-//import uk.ac.ebi.atlas.model.experiment.differential.DifferentialExperiment;
-//import uk.ac.ebi.atlas.model.experiment.differential.microarray.MicroarrayExperiment;
-//import uk.ac.ebi.atlas.resource.ContrastImageSupplier;
-//import uk.ac.ebi.atlas.trader.ExperimentTrader;
-//
-//import javax.inject.Inject;
-//import javax.inject.Named;
-//import javax.servlet.http.HttpServletResponse;
-//import java.net.URI;
-//import java.util.List;
-//import java.util.function.Function;
-//
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.servlet.http.HttpServletResponse;
+import java.net.URI;
+import java.util.List;
+import java.util.function.Function;
 @Named
 public class SingleCellContentService {
```

Comment:
I couldn’t see this class being used anywhere. Is it used in any of the two sub-branches?

Reply:
Yes, it's used later on in the Supplemetary Information tab but I think it could be refactored in the future...